docs/guides/security-guide.md

```diff
@@ -132,18 +119,25 @@ by the ca.crt (can copy the generated files from the server if client is not
 on the same system as server).
 
 ### P4RT client
-
-The p4rt-ctl (P4RT client) will default to communicate via secure mode
-(port 9559). If certificates are not available, the P4RT client will attempt a
+The p4rt-ctl (P4RT client) will default to communicate via secure mode using
+port 9559. If certificates are not available, the P4RT client will attempt a
 connection using insecure client credentials as a fallback mechanism. Note that
-the communication may fail if infrap4d runs in secure mode only.
+the communication may fail if infrap4d runs in secure mode only. Both server and
+client should specify insecure mode for this to work.
 
 ### gNMI client
 
-The gnmi-ctl or sgnmi_cli (gNMI clients) requests should be directed to port 9339.
-If infrap4d is enabled to run in insecure mode, a flag is available on client side
+The gnmi-ctl(gNMI client for DPDK) or sgnmi_cli (gNMI client for Intel IPU E2100) requests
```

Comment:
Unfortunately, the cool markup feature doesn't work (or I can't figure out how to make it work) on multiple lines.

"gnmi-ctl (the gNMI client for DPDK) and sgnmi_cli (the secure gNMI client) issue requests to port 9339. If infrap4d is started in insecure mode, a flag must be specified so the client also runs in insecure mode."